### PR TITLE
Add micaela/langtonant to dev/langtonant branch

### PR DESCRIPTION
**Pull request by micakler**

### CODE REVIEW

File: katas/langtonant/micaela/functions.tests.js
Comment (micakler) [inline katas/langtonant/micaela/functions.tests.js:37]: Isn´t finished

Comment (micakler) [inline katas/langtonant/micaela/functions.tests.js:37]: Isn´t finished

File: katas/langtonant/micaela/functions.tests.js
Comment (J0c4) [inline katas/langtonant/micaela/functions.tests.js:1]: Looks like the tests are not being run because file name contains `tests` instead of `test`, please fix the file name so the tests can be run

File: package.json
Comment (J0c4) [inline package.json:7]: script is ok, but micaela/main.js is missing, still pending to be added?

**Review by J0c4 — CHANGES_REQUESTED** [review]
Hi Micaela, looks like you have good progress here, but still need to complete the code to test everything and run main code, please include missing changes

**Review by J0c4 — CHANGES_REQUESTED** [review]
Hi @micakler , you have good progress here. Few things that need to be complete before merge:
- there are some lint issues, please fix them
- there is a conflict that needs to be solved, you can sync with @AlfredPereira and @guillav29 to see who will merge his PR first so you guys don't need to solve conflicts two or three times again.
- When I run main, there seems to be an error importing a dependency, maybe can be fixed if you add '.js' extension to the file referenced in the import

**Comment by J0c4** [comment]
Hi Micaela, thanks for the updates, there are couple that still need to be fixed:
- Error while running main.js
- please revert changes commited for package-lock.json file

**Comment by J0c4** [comment]
for reverting package-lock.json file, you can try this: https://dev.to/lofiandcode/git-and-github-how-to-revert-a-single-file-dha

**Comment by J0c4** [comment]
Hi @micakler , any updates on this one? if issues do not get fixed, we cannot merge this as it will break the build status. Since its been a while for this PR, we will need to set tomorrow as a deadline so we can close it

**Comment by micakler** [comment]
I tried to revert package-lock.json file but i couldn't do it

**Comment by J0c4** [comment]
Ok, lets close this one for now. We can follow up in case you have further questions about this one.

**Pull request closed without merging**
